receiver/jmxreceiver/config.go

```diff
@@ -197,7 +254,8 @@ func (c *Config) validateJar(supportedJarDetails map[string]supportedJar, jar st
 }
 
 var (
-	validLogLevels     = map[string]struct{}{"trace": {}, "debug": {}, "info": {}, "warn": {}, "error": {}, "off": {}}
+	validLogLevels = map[string]struct{}{"trace": {}, "debug": {}, "info": {}, "warn": {}, "error": {}, "off": {}}
+	// feature parity between jmx-gatherer and jmx-scraper
```

Comment:
This list is expected to change over time so maybe removing this hard-coded list might be a better option (both scraper and gatherer fail when we try to use a non-existing target system).

In we keep an exhaustive list, adding a link to the supported systems in scraper might be a good idea to make maintenance easier: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-scraper/README.md#configuration-reference

Reply:
Good point, issue url: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41104